packages/runtime-dom/src/modules/events.ts

```diff
@@ -109,11 +109,22 @@ function createInvoker(
       [e]
     )
   }
-  invoker.value = initialValue
+  invoker.value = sanitizeEventValue(initialValue)
```

Comment:
I've created an example to illustrate what I mean:

[SFC Playground example](https://deploy-preview-8953--vue-sfc-playground.netlify.app/#eNqNUsFO4zAQ/ZWRLy1S1arlFqWVdhGHXWkXBEhcfAnutJg648gel6Iq/87EgVAhhDglnjfvzfPzHNWvppnuE6pCldEE2zBE5NSsNNm68YHhCAErw3aPE/D0zydiXEMLm+BrGAl1pEmT8RQZHitaOwwRlgNpfNQE78C8gBFj5PloclJdFDA+g+UKjq2m9qyTGwaN35CT9jh958mYLLcQCx2vnPVXEPNyYKwbVzHKCaB8SMyeoPB04azZLbX6LDfXanVfBbK0hWcfdhGMDwENu5dy1tN/KrUQqVu2zsEWOYIl9lCRfDeWLKOEY1KI1ovjQbicDYbVRHGURDd2O32KnuRxcgBaGV83VgZcNSzsqFXRR9NhlXP++W+ucUiYE86cRzS7L+pP8dDVtLoOGDHsUasB4yqI8x6+vP2PB/kfwNqvk5Pub8AbjN6lzmPf9jvlXE76sts/ecUk8Lt4eWCkLpLBaNfZ5n6tZM0uvrn6h93z6XnmyUKo9hXetvmR)

The warning is being shown for the first button but not for the second button, because only the initial value is being checked.

Reply:
@skirtles-code Ah, gotcha I'll see to move it to patchEvent